src/dune_rules/odoc.ml

```diff
@@ -371,15 +373,17 @@ let setup_html sctx (odoc_file : odoc_artefact) =
 let setup_css_rule sctx =
   let open Memo.O in
   let ctx = Super_context.context sctx in
+  let dir = Paths.html_root ctx ++ "_odoc_support" in
   let* run_odoc =
     run_odoc sctx ~dir:(Path.build ctx.build_dir) "support-files"
-      ~flags_for:None
-      [ A "-o"
-      ; Path (Path.build (Paths.html_root ctx))
-      ; Hidden_targets [ Paths.css_file ctx; Paths.highlight_pack_js ctx ]
-      ]
+      ~flags_for:None [ A "-o"; Target dir ]
   in
-  add_rule sctx run_odoc
+  add_rule sctx
+    (Action_builder.progn
+       [ Action_builder.with_no_targets
+           (Action_builder.return (Action.Full.make (Action.Remove_tree dir)))
```

Comment:
I'm not sure this is the right sort of thing to do here. It's modelled on the `html-generate` rule immediately above.

Reply:
what happens if the `Remove_tree` part is removed?

Reply:
If anything else was in there we'd end up leaving it there. Not a big deal I guess? I'm not quite clear on when this rule is executed. It certainly seems to be the case that if I remove individual files from within the `_odoc_support` dir they get put back next time I do `dune build @doc`